Sources/SwiftIDEUtils/SyntaxClassifier.swift

```diff
@@ -196,12 +196,49 @@ private struct ClassificationVisitor {
 
     for case (let index, let child?) in children.enumerated() {
 
-      let classification: (SyntaxClassification, Bool)?
+      let classification: (classification: SyntaxClassification, force: Bool)?
       if case .layout(let layout) = descriptor.node.kind.syntaxNodeType.structure {
         classification = SyntaxClassification.classify(layout[index])
       } else {
         classification = nil
       }
+
+      if let classification, classification.force {
+        // Leading trivia.
+        if let triviaPieces = child.leadingTriviaPieces {
+          for triviaPiece in triviaPieces {
+            let range = triviaPiece.classify(offset: byteOffset)
+            report(range: range)
+            byteOffset += triviaPiece.byteLength
+          }
+        }
+        child.leadingTriviaPieces?.compactMap { $0 }.forEach {
+          let range = $0.classify(offset: byteOffset)
+          report(range: range)
+        }
```

Comment:
I think you’re classifying `leadingTriviaPieces` twice: Once in the `for` loop above and then once here.